src/components/layouts/FilterBar.tsx

```diff
@@ -268,7 +269,7 @@ export const FilterBar = ({ mode, updateViewModeSetting }: FilterBarProps) => {
               }}
             />
 
-            <ViewModeSelector
+            <DisplaySelector
```

Comment:
Remove the `ViewModeSelector` since it is not used anymore
![image](https://github.com/user-attachments/assets/1cbde100-43a9-48d4-a59c-ab73a21ca930)